### PR TITLE
Portfolio popup window

### PR DESCRIPTION
**Pull request by tinatsina**
Used pair programming in order to solve and place modal windows

## JS Object Approach
- Used JS nested objects to store the data
- All contents of the Modal where dynamically loaded from JS Object

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,9 +87,57 @@ <h4 class="connect-text">LET'S CONNECT</h4>
...
+            <button type="submit" class="see-project see-project-padding modal-button">
+                See live <img src="img/21.svg" alt="">
+            </button>
+            <button type="submit" class="see-project see-project-padding modal-button">
+              See Soure <img src="img/22.svg" alt="">
+          </button>
+        </div>
+    </div>
+    </div>
+    <div id="overlay"></div>
```
Comment (muneebulrehman) [inline index.html:135]: The position of your close button on the desktop screen is not as per the Figma template. Consider fixing it.
image

File: index.html [hunk 2]
```diff
@@ -87,9 +87,57 @@ <h4 class="connect-text">LET'S CONNECT</h4>
...
+
     <!-- PORTFOLIO SECTION -->
 
     <section id="the-portfolio-section">
```
Comment (muneebulrehman) [inline index.html:139]: All your portfolio should cards should be generated using javascript. Kindly remove them from your HTML file and use javascript to generate the cards.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @tinatsina , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tinatsina) [reply index.html]: ## Resolved 👍 
- used `min-width: 768px` to adjust how the button appears in desktop mode
- Also made minor touchups
- Thanks for the assist

File: index.html (hunk 2 above)
Comment (tinatsina) [reply index.html:133]: ## Resolved 👍 
- Emptied the HTML modal elements
- Hence they are only loaded from the JS nested object
- All data comes from `modalObject`
- Thanks for the assist

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**